src/Makefile.in

```diff
@@ -0,0 +1,32 @@
+CC ?= gcc
+LUAJIT_FLAGS = $(shell pkg-config --cflags --libs luajit)
+LUAJIT_VER = $(shell pkg-config --variable=abiver luajit)
+LUASRC = $(shell find . -regex '[^\#]*\.lua' -printf '%P ')
+
+
+BINDIR = $(DESTDIR)/@DYZBINDIR@
+LUADIR = $(DESTDIR)/@DYZLUADIR@
+
+dyz: dyz.c config.h
+	$(CC)  -o $@ $? $(LUAJIT_FLAGS) -Wl,--no-as-needed -Wl,-E
+
+install-bin: dyz
+	@mkdir -p $(BINDIR)
+	@cp $^ $(BINDIR)
+
+copy-luafiles: $(LUASRC)
```

Comment:
If the Lua sources get linked into `dyz` from `.o` files containing bytecode, there is no need to install the sources, and this can be removed.